src/frequenz/client/dispatch/__main__.py

```diff
@@ -171,11 +171,26 @@ def format_line(key: str, value: str, color: str = "cyan") -> str:
     show_envvar=True,
 )
 @click.option(
-    "--key",
-    help="API key for authentication",
+    "--api-key",
+    help="API key for authentication (deprecated, use DISPATCH_API_AUTH_KEY)",
```

Comment:
Ah, nice, maybe include `--api-key` too.
```suggestion
    help="API key for authentication (deprecated, use --auth-key or DISPATCH_API_AUTH_KEY instead)",
```